pkg/network/ovn_kubernetes.go

```diff
@@ -456,18 +459,21 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 
 	manifestDirs = append(manifestDirs, filepath.Join(manifestSubDirBasePath, "common"))
 
-	// choose the YAMLs based on the target zone mode (4.14 only) TODO: starting from 4.15, support multizone only
-	manifestSubDir := filepath.Join(manifestSubDirBasePath, "multi-zone-interconnect") // default is multizone
+	ongoingMigrationFromSingleToMultiZone := (!targetZoneMode.fastForwardToMultiZone &&
```

Comment:
```suggestion
	ongoingUpgradePhaseFromSingleToMultiZone := (!targetZoneMode.fastForwardToMultiZone &&
```
I think "migration" is too overloaded, I'd prefer to keep all the upgrade related logic as some sort of upgrade related naming for better readability. (mtu migration, cni live/offline migration, gateway mode migration etc etc)

Reply:
It was a handy word ;)